rasa/core/events/__init__.py

```diff
@@ -735,6 +748,16 @@ def __eq__(self, other):
     def __str__(self):
         return "StoryExported()"
 
+    @classmethod
+    def _from_story_string(cls, parameters: Dict[Text, Any]) -> Optional[List[Event]]:
+        return [
+            StoryExported(
+                parameters.get("path"),
+                parameters.get("timestamp"),
+                parameters.get("metadata"),
```

Comment:
could we use the default implementation of the `Event` parent class?

Reply:
If we never expect `_from_story_string` to be called for a `StoryExported` event, then yes (otherwise we'd be losing the "path" property).